spring-cloud-modules/spring-cloud-aws-v3/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.spring.cloud</groupId>
+    <artifactId>spring-cloud-aws-v3</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>spring-cloud-aws-v3</name>
+    <packaging>jar</packaging>
+    <description>Spring Cloud AWS Examples</description>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.awspring.cloud</groupId>
+                <artifactId>spring-cloud-aws</artifactId>
+                <version>3.1.0</version>
```

Comment:
We have a convention to keep all dependency versions in a property, can you extract a property for this version number

Reply:
Just a note - I was having lots of troubles with dependency versions when inheriting the parent project's pom since it uses fairly outdated dependencies, so I let go of that inheritance and created an independent pom.xml. It still compiles as part of the same project though.

Perhaps we could consider having a separate spring cloud module for Spring Boot 3 and adding new tutorials there.

Reply:
Hmm, yes I'll ask internally about what the best option would be here